src/css/properties/css_modules.zig

```diff
@@ -100,17 +109,28 @@ pub const Specifier = union(enum) {
     /// The referenced name is global.
     global,
     /// The referenced name comes from the specified file.
-    file: []const u8,
+    ///
+    /// Is an import record index
+    file: u32,
```

Comment:
Please call this `import_record_index` instead of `file` so that we are consistent with the rest of the codebase.